iib/web/api_v1.py

```diff
@@ -439,21 +444,38 @@ def regenerate_bundle_batch():
     # This list will be used for the log message below and avoids the need of having to iterate
     # through the list of requests another time
     request_id_strs = []
```

Comment:
Now that this being used for things other than log messages, it might be better to rename this variable and not have it contain strings:
```suggestion
    processed_request_ids = []
```